apps/src/applab/designMode.js

```diff
@@ -508,7 +523,9 @@ function makeDraggable (jqueryElements) {
 
     wrapper.css({
       top: elm.css('top'),
-      left: elm.css('left')
+      left: elm.css('left'),
+      width: parseInt(elm[0].style.width, 10) + parseInt(elm[0].style.margin, 10),
+      height: parseInt(elm[0].style.height, 10) + parseInt(elm[0].style.margin, 10)
```

Comment:
the only reason for margins that I know of is to prevent elements dynamically generated by textLabel(), write(), and other applab commands from appearing flush against the left (or top) of the applab frame. design-mode-generated elements shouldn't need any margin.